src/docs/markdown/service.md

```diff
@@ -0,0 +1,121 @@
+
+You can stop the service with:
+<pre><code class="cmd bash">sudo systemctl stop caddy</code></pre>
+
+<aside class="advice">
+	Do not stop the service to change Caddy's configuration. Stopping the server will incur downtime. Use the reload command instead.
+</aside>
+
+
+## Manual Installation
+
+Some [installation methods](/docs/install) automatically set up Caddy to run as a service. If you chose a method that did not, you may follow these instructions to do so:
+
+**Requirements:**
+
+- `caddy` binary that you [downloaded](/download) or [built from source](/docs/build)
+- `systemctl --version` 232 or newer
+- `sudo` privileges
+
+Move the caddy binary into your `$PATH`, for example:
+<pre><code class="cmd bash">sudo mv caddy /usr/bin/</code></pre>
+
+Test that it worked:
+<pre><code class="cmd bash">caddy version</code></pre>
+
+Create a group named `caddy`:
+<pre><code class="cmd bash">sudo groupadd --system caddy</code></pre>
+
+Create a user named `caddy`, with a writeable home directory:
+<pre><code class="cmd bash">sudo useradd --system \
+    --gid caddy \
+    --create-home \
+    --home-dir /var/lib/caddy \
+    --shell /usr/sbin/nologin \
+    --comment "Caddy web server" \
+    caddy</code></pre>
+
+If using a config file, be sure it is readable by the `caddy` user you just created.
+
+Next, [choose a systemd unit file](#unit-files) based on your use case.
+
+**Double-check the `ExecStart` and `ExecReload` directives.** Make sure the binary's location and command line arguments are correct for your installation! For example: if using a config file, change your `--config` path if it is different from the defaults.
+
+The usual place to save the service file is: `/etc/systemd/system/caddy.service`
+
+After saving your service file, you can start the service for the first time with the usual systemctl dance:
+
+<pre><code class="cmd"><span class="bash">sudo systemctl daemon-reload</span>
+<span class="bash">sudo systemctl enable caddy</span>
+<span class="bash">sudo systemctl start caddy</span></code></pre>
+
+Verify that it is running:
+<pre><code class="cmd bash">systemctl status caddy</code></pre>
+
+Now you're ready to [use the service](#using-the-service)!
+
+
+## Overrides
+
+The best way to override aspects of the service files is with this command:
+<pre><code class="cmd bash">sudo systemctl edit caddy</code></pre>
+
+This will open a blank file with your default terminal text editor in which you can override or add directives to the unit definition.
+
+For example, if you need to define environment variables for use in your config, you may do so like this:
+```systemd
+[Service]
+Environment="CF_API_TOKEN=super-secret-cloudflare-tokenvalue"
+```
+
+Or, for example if you need to change the config file from the default of the Caddyfile, to instead using a JSON file:
+```systemd
+[Service]
+ExecStart=/usr/bin/caddy run --environ --config /etc/caddy/caddy.json
+ExecReload=/usr/bin/caddy reload --config /etc/caddy/caddy.json
```

Comment:
`Exec*` directives are allowed to be specified multiple times.  An override with just another `Exec*` line will cause both the original and the new one to be run.  They has to be "reset" with empty directives to properly override them.

https://www.freedesktop.org/software/systemd/man/systemd.service.html#ExecStart=

```suggestion
ExecStart=
ExecStart=/usr/bin/caddy run --environ --config /etc/caddy/caddy.json
ExecReload=
ExecReload=/usr/bin/caddy reload --config /etc/caddy/caddy.json
```

Reply:
TIL! Thanks